scrapy/item.py

```diff
@@ -24,18 +24,20 @@ class Field(dict):
 class ItemMeta(ABCMeta):
 
     def __new__(mcs, class_name, bases, attrs):
+        _class = super(ItemMeta, mcs).__new__(mcs, 'x_' + class_name, tuple(base._class for base in bases if hasattr(base, '_class')), attrs)
```

Comment:
hey @nramirezuy - could you please split it into multiple lines? 
e.g. `tuple(base._class for base in bases if hasattr(base, '_class'))` is a good candidate for a local variable.